proto/perp/amm/v1/state.proto

```diff
@@ -145,7 +145,19 @@ message ReserveSnapshot {
   ];
 
   // milliseconds since unix epoch
-  int64 timestamp_ms = 3;
```

Comment:
Do you know why there is this change in number? In theory this is not recommended.

Reply:
3 and 5 are missing field numbers.